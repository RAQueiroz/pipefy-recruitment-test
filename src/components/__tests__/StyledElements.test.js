import React from "react";
import {
  Button,
  Card,
  ErrorMessage,
  InlineError,
  Spacer,
  SuccessMessage
} from "../StyledElements";
import { render } from "react-testing-library";

describe("Test snapshots of styled elements", () => {
  const { container: buttonContainer } = render(<Button />);
  const { container: cardContainer } = render(<Card />);
  const { container: errorMessageContainer } = render(<ErrorMessage />);
  const { container: inlineErrorContainer } = render(<InlineError />);
  const { container: spacerContainer } = render(<Spacer />);
  const { container: successMessageContainer } = render(<SuccessMessage />);

  it("Button to match snapshot", () => {
    expect(buttonContainer.firstChild).toMatchSnapshot();
  });

  it("Card to match snapshot", () => {
    expect(cardContainer.firstChild).toMatchSnapshot();
  });

  it("ErrorMessage to match snapshot", () => {
    expect(errorMessageContainer.firstChild).toMatchSnapshot();
  });

  it("InlineError to match snapshot", () => {
    expect(inlineErrorContainer.firstChild).toMatchSnapshot();
  });

  it("Spacer to match snapshot", () => {
    expect(spacerContainer.firstChild).toMatchSnapshot();
  });

  it("SuccessMessage to match snapshot", () => {
    expect(successMessageContainer.firstChild).toMatchSnapshot();
  });
});
