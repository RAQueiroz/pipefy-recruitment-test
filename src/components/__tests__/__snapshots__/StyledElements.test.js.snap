// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test snapshots of styled elements Button to match snapshot 1`] = `
<button
  class="sc-bdVaJa isjKgN"
/>
`;

exports[`Test snapshots of styled elements Card to match snapshot 1`] = `
<div
  class="sc-bwzfXH kgKear"
/>
`;

exports[`Test snapshots of styled elements ErrorMessage to match snapshot 1`] = `
<div
  class="sc-htpNat hCJIlZ"
/>
`;

exports[`Test snapshots of styled elements InlineError to match snapshot 1`] = `
<div
  class="sc-ifAKCX kKLNNA"
/>
`;

exports[`Test snapshots of styled elements Spacer to match snapshot 1`] = `
<div
  class="sc-EHOje cAQQWe"
/>
`;

exports[`Test snapshots of styled elements SuccessMessage to match snapshot 1`] = `
<div
  class="sc-bxivhb bCAxHu"
/>
`;
